mod-circulation/src/main/resources/domain/mod-circulation/features/loans.feature

```diff
@@ -216,3 +217,35 @@ Feature: Loans tests
     Then status 200
     Then match $.loans[0].dueDateChangedByRecall == true
     And match $.loans[0].dueDate !contains expectedDueDateBeforeRequest
+
+  Scenario: When an requested loaned item is checked in at a service point designated as the pickup location of the request, change the item status to awaiting-pickup
+
+    * def extItemBarcode = '' + random(100000)
```

Comment:
Why is it random? Can we use a constant here?

Reply:
because last PR, Victor suggested me to use random function